### PR TITLE
Add ECKeypair doc for :model module

### PR DESCRIPTION
**Pull request by bigman212**
I've decided to help and write some documentation on how to use this lib. 
I haven't done much here - I want to ensure this is what you want. 

https://dillinger.io/ - to preview readme

### CODE REVIEW

File: wallet/src/main/kotlin/org/kethereum/wallet/WalletFile.kt [hunk 1]
```diff
@@ -47,9 +47,11 @@ internal val moshi by lazy {
 }
 
 @Throws(CipherException::class, IOException::class)
-fun ECKeyPair.generateWalletFile(password: String,
-                                                     destinationDirectory: File,
-                                                     config: ScryptConfig) = createWallet(password, config).let { wallet ->
+fun ECKeyPair.generateWalletFile(
+    password: String,
```
Comment (bigman212) [inline wallet/src/main/kotlin/org/kethereum/wallet/WalletFile.kt:51]: sorry, but these indents were something else :)

File: wallet/src/test/kotlin/org/kethereum/wallet/WalletTest.kt [hunk 2]
```diff
@@ -10,36 +10,42 @@ class WalletTest {
...
+        val (keyPairFromFile) = walletFile.toTypedWallet().decrypt(PASSWORD)
+        assertThat(keyPairFromFile.key.toHexStringNoPrefix()).isEqualTo(PRIVATE_KEY_STRING)
     }
 
     @Test
     fun testDecryptScrypt() {
-        val walletFile = load(SCRYPT_TEST_JSON)
-        val (privateKey) = walletFile.toTypedWallet().decrypt(PASSWORD)
-        assertThat(privateKey.key.toHexStringNoPrefix()).isEqualTo(PRIVATE_KEY_STRING)
+        val walletFile = loadWalletFile(SCRYPT_TEST_JSON)
```
Comment (bigman212) [inline wallet/src/test/kotlin/org/kethereum/wallet/WalletTest.kt:46]: all unit tests are green, just a small variable extraction to make it more readable

File: model/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+This module contains base classes that are used for working with KEthereum
+
+### Installation
+[![](https://jitpack.io/v/komputing/kethereum.svg)](https://jitpack.io/#komputing/kethereum)
+```groovy
+implementation("com.github.walleth.kethereum:model:$KETHEREUM_VERSION")
```
Comment (ligi) [inline model/README.md:6]:
```suggestion
implementation("com.github.komputing.kethereum:model:$KETHEREUM_VERSION")
```

**Review by ligi — CHANGES_REQUESTED** [review]
Thanks! Just minor nits

File: model/README.md (hunk 3 above)
Comment (bigman212) [reply model/README.md]: good one, thanks!

**Comment by bigman212** [comment]
@ligi feel free to merge.

**Review by ligi — APPROVED** [review]

**Pull request merged**